src/seals/diagnostics/early_term.py

```diff
@@ -0,0 +1,58 @@
+"""Environment checking for correctness under early termination."""
+
+from functools import partial
+
+import numpy as np
+
+from seals import base_envs
+
+
+class EarlyTerminationEnv(base_envs.TabularModelPOMDP):
+    """Three-state MDP with early termination state.
+
+    Many implementations of imitation learning algorithms incorrectly assign a
+    value of zero to terminal states (Kostrikov et. al (2018)).  Depending on
```

Comment:
Can we add a clearer citation? https://scholar.google.com/scholar?hl=en&as_sdt=0%2C5&q=kostrikov+2018&btnG= shows multiple papers written by a Kostrikov in 2018.